PolishWithBlondes/src/components/navbar/navbar.scss

```diff
@@ -0,0 +1,42 @@
+.navbar {
+  background-color: var(--primary-background);
+  color: var(--primary-font-color);
+  font-family: var(--font-family);
+  font-size: 14px;
+  padding: 40px;
+  display: flex;
+  justify-content: space-between;
+  max-width: 1060px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
```

Comment:
that is a default value, no need to set it

Reply:
resolved